porch/internal/tools/tools.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build tools
+// +build tools
+
+package tools
+
+import (
+	_ "golang.org/x/tools/cmd/stringer"
+)
```

Comment:
Why is this `tools` package needed ?

Reply:
It allows us to use the import without go complaining about it during a regular build. We already have a similar pattern in use the `porch/api` module. 
This is needed as a result of https://github.com/GoogleContainerTools/kpt/pull/3577.